### PR TITLE
Blogspot source

### PR DESCRIPTION
**Pull request by markbaas**
- ~~fixes proxydb~~
- adds support for proxy lists in blogspot.

### CODE REVIEW

**Comment by chill117** [comment]
Looks like a test is failing because you need to update the [sample page HTML for proxydb](https://github.com/chill117/proxy-lists/tree/master/test/samples/proxydb/pageHtml).

**Comment by chill117** [comment]
I've included the proxydb fixes from this PR. If you'd like to continue with the blogspot source, let me know. If I have time, I'll look at adding it as well.

**Comment by markbaas** [comment]
Okay thanks! I'm little on time atm. How would you like to combine the blogspot sources? I port the code from the python proxybroker which was die hard matching ips from the dom. I see the proxies24 plugin is a bit more extensive.

File: sources/blogspot.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+			cb(null, data);
+		});
+	},
+  	parseListHtml: function(listHtml, cb) {
+        var proxies = [];
+
+        var reg = /\d+\.\d+\.\d+\.\d+\:\d+/g
+        var matches = [], found;
+        while (found = reg.exec(listHtml)) {
```
Comment (chill117) [inline sources/blogspot.js]: I don't think it's a good idea to use regular expressions on large chunks of text. Could this use an HTML parser (cheerio) or RSS/XML parser instead?

File: sources/blogspot.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+		});
+
+		return emitter;
+    },
+    prepareListUrls: function (options) {
+        return ['sslproxies24.blogspot.com', 'proxyserverlist-24.blogspot.com',
+            'newfreshproxies24.blogspot.com', 'irc-proxies24.blogspot.com',
+            'freeschoolproxy.blogspot.com', 'googleproxies24.blogspot.com',
+            'getdailyfreshproxy.blogspot.com']
+    },
```
Comment (chill117) [inline sources/blogspot.js]: Looks like some of these URLs are no longer active blogs.

**Comment by chill117** [comment]
It looks like you already have some of the proxies24 URLs. So, as long as all the existing URLs from that source are included, I think that covers it.

**Comment by chill117** [comment]
The idea of this PR is good. There are some sources hosted on blogspot that could all be scraped in the same way, because the HTML structures of the sources are similar enough to parse in the same way.

**Comment by markbaas** [comment]
I have been testing this for a while. The regexes are too heavy. You already included in most important sources, however I think you are still missing some ips, it would be better to fix the existing source reading for rss instead of the html posts.

**Comment by chill117** [comment]
I can re-visit this to create a blogspot source that consolidates all the current proxy sources that come from blogspot pages.

**Pull request closed without merging**